### PR TITLE
feat(post): Move post domain beneath post title

### PR DESCRIPTION
**Pull request by jsit**
Some design work here, trying to tidy up all the metadata grouped together in the post details.

I've moved the link domain and the "expand" button to beneath the post title in small text. I think this makes it easier to parse the information.

## Before

Screenshot 2023-06-18 at 10 37 24 AM
Screenshot 2023-06-18 at 10 37 19 AM

## After
Screenshot 2023-06-18 at 2 33 04 PM
Screenshot 2023-06-18 at 2 33 12 PM

### CODE REVIEW

**Comment by glibg10b** [comment]
It would be more intuitive if the preview button's behavior swapped with the thumbnail's behavior:
- The button next to the domain would link to the specific page at the domain (and have the appropriate icon)
- The post's thumbnail would then show the preview when clicked, which would make it consistent with image posts, which already have that behavior

This would solve #1142 and #634

If you don't think this should be addressed in this PR, I will start a new issue

**Comment by jsit** [comment]
@glibg10b Hm yeah I think this is a change in behavior that's outside the scope of this PR. Thank you though!

Though I will say that link posts that have thumbnails have *both* (a) expandable images and (b) expandable text. This is from beehaw.org which I think runs 0.17.4:

Screenshot 2023-06-18 at 1 39 04 PM

File: src/assets/css/main.css [hunk 1]
```diff
@@ -84,10 +84,6 @@
   margin-top: -6.5px;
 }
 
-.post-title {
-  line-height: 1;
-}
```
Comment (dessalines) [inline src/assets/css/main.css:89]: Thanks. Every bit of custom css we can remove is a win.

**Review by dessalines — APPROVED** [review]
Looks much better, thanks.

**Pull request merged**